### PR TITLE
add update link functions

### PR DESCRIPTION
**Pull request by LucaV27**
I needed to implement the function of updating a GNS3 link.
Especially I used it for the "suspend" and "filters" fields of the input, so I added a minimum validator in the code for previously mentioned fields.
All references from:
http://api.gns3.net/en/2.2/api/v2/controller/link/projectsprojectidlinkslinkid.html#put-v2-projects-project-id-links-link-id

### CODE REVIEW

**Comment by LucaV27** [comment]
Any news on this?

File: gns3fy/gns3fy.py [hunk 1]
```diff
@@ -1758,17 +1793,11 @@ def create_link(self, node_a, port_a, node_b, port_b):
                     node_id=_node_a.node_id,
                     adapter_number=_port_a["adapter_number"],
                     port_number=_port_a["port_number"],
-                    label=dict(
-                        text=_port_a["name"],
-                    ),
                 ),
```
Comment (davidban77) [inline gns3fy/gns3fy.py:1796]: Is there a particular reason the label needs to be deleted?

**Review by davidban77 — CHANGES_REQUESTED** [review]
Thank you for your PR.

The `update` method for the links is already on the `develop` branch and I am planning on a release soon.

Would you mind making the PR against the `develop` branch? I believe the extra validations you have for the other Link attributes are useful to have.

Thanks!

File: gns3fy/gns3fy.py (hunk 1 above)
Comment (LucaV27) [reply gns3fy/gns3fy.py:1796]: Regarding this, this change was done by mistake (actually I don't know how) and I had no intention to change this code.

**Comment by LucaV27** [comment]
Okay perfect. I close this PR and will create a new one to the `development` branch next week. I will not push the change you mention in the above comment.
Many thanks.

**Pull request closed without merging**